tools/license2rtf.js

```diff
@@ -1,15 +1,16 @@
+'use strict';
 
 var assert = require('assert'),
-    Stream = require('stream'),
-    inherits = require('util').inherits;
+  Stream = require('stream'),
+  inherits = require('util').inherits;
```

Comment:
Not sure.
Perhaps this should use separate declarations and `const` instead, if it's already being changed?
i.e.
```js
const assert = require('assert');
const Stream = require('stream');
const inherits = require('util').inherits;
```

The same for other variable declarations changed by this commit. Thoughts?